CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 ## Unreleased
 
 ### Features
-...
+- Add Catalan and Polish support to `detectLanguage`.
```

Comment:
detectLanguage seems to have weird casing, are you sure that its called like this?